README.md

```diff
@@ -96,18 +90,18 @@ To keep from having to install files to the user's system, the preferred method
 ```console
 $ echo "PATH=<LocationOfDUNE>:$PATH" >> .bashrc
```

Comment:
Is it possible to add a script in the deb package which will add it to the PATH during the installation?

Reply:
Yes, on Windows choco will modify environment variable %PATH% using this command: https://github.com/AntelopeIO/DUNE/pull/94/files#diff-2be06024332f8d9ce2e847cc5c172d6f2255afabacc3433989507c8379942f06R5
On Linux the link `antelopeio-dune` is created in /usr/bin/ using this command: https://github.com/AntelopeIO/DUNE/blob/main/packaging/generate_tarball.sh#L20

On Linux there is a name clash with `dune` command which is installed by default i.e. in Ubuntu, that's why the name `antelopeio-dune`.